scripts/run_integration_tests.sh

```diff
@@ -29,4 +30,4 @@ docker exec -it qgis sh -c "git clone -q -b $BRANCH --depth=1 https://github.com
 
 docker exec -it qgis sh -c "qgis_setup.sh svir"
 
-docker exec -it qgis sh -c "cd /tests_directory && SELECTED_CALC_ID=$SELECTED_CALC_ID qgis_testrunner.sh svir.test.integration.test_drive_oq_engine"
```

Comment:
Just curious, why `SELECTED_CALC_ID=$SELECTED_CALC_ID` disappeared?

Reply:
Yes, that's why